packages/docs/base/color.md

```diff
@@ -250,6 +250,23 @@ Here's a quick overview of the colors available in Odyssey and some guidelines f
           N/A
         </td>
       </tr>
+      <tr>
+        <td class="is-sample-table-swatch">
+          <span class="sample-color sample-color--green-600"></span>
+        </td>
+        <td class="is-sample-table-token">
+          Green 600
+        </td>
+        <td>
+          <code>#008055</code>
+        </td>
+        <td>
+          Background color for success messaging
+        </td>
+        <td>
+          Headlines, body copy, fail states, buttons
+        </td>
+      </tr>
```

Comment:
These changes are not visible [here](https://d259960.ods.dev/base/color/). Any idea why?

Reply:
This is the correct preview: https://540eef8.ods.dev/base/color/